src/main/java/com/github/princesslana/greedorama/commands/Format.java

```diff
@@ -18,6 +21,33 @@ public static String money(MonetaryAmount amt) {
     return fmt.format(amt);
   }
 
+  public static String date(Instant inst) {
+    var fmt =
+        DateTimeFormatter.ofPattern("MMM dd uuuu")
+            .withLocale(Locale.getDefault())
+            .withZone(ZoneId.systemDefault());
+
+    return fmt.format(inst);
+  }
+
+  public static String shortDate(Instant inst) {
+    var fmt =
+        DateTimeFormatter.ofPattern("uuuu-MM-dd")
+            .withLocale(Locale.getDefault())
+            .withZone(ZoneId.systemDefault());
+
+    return fmt.format(inst);
+  }
+
+  public static String time(Instant inst) {
```

Comment:
Now we don't display time, this is unused?

Reply:
Yes, I just left it in case time is added back in to the history command. I can delete it.